### ISSUE TITLE
Soap Error: looks like we got no XML document

### ISSUE THREAD

**Issue by bhenstra** [w2c/ispconfig3_roundcube#101]
Hello,
Thank you for the plugin. It worked fine for me; but after upgrading from Debian Stretch to Buster I keep getting this error: "Soap Error: looks like we got no XML document"
I could not find a solution. Could you please help? Thanks.
Bouke

**Comment by Jemt** [comment]
You may have to reinstall the plugins after upgrading Debian since upgrades may wipe manually installed files from the system. While I'm not sure whether that's the problem in you case, it's worth a shot.

**Comment by bhenstra** [comment]
> You may have to reinstall the plugins after upgrading Debian since upgrades may wipe manually installed files from the system. While I'm not sure whether that's the problem in you case, it's worth a shot.

Thank you for your reply - very much appreciated. 

I did re-install already. I did download the master.zip and copied the files into /var/lib/roundcube/plugins - overwriting the files. I did verify the file "ispconfig_account/config/config.php" and it looks just fine.

I did also verify /etc/roundcube/config.inc.php and it looks fine too:

> // List of active plugins (in plugins/ directory)
> // Debian: install roundcube-plugins first to have any
> $config['plugins'] = array(
> "jqueryui","ispconfig3_account", "ispconfig3_autoreply",
> "ispconfig3_pass", "ispconfig3_spam",
> "ispconfig3_fetchmail", "ispconfig3_filter",
> "ispconfig3_forward", "ispconfig3_wblist",
> "twofactor_gauthenticator","rcguard","mobile"
> );

I don't know where to look next. Please help. Thanks :-)

**Comment by Churro** [comment]
Please ensure that the SOAP module for PHP is still installed. When upgrading from stretch to buster, Debian's default PHP version changes from 7.0 to 7.3. So you could try: `apt install php7.3-soap`

You could also call the SOAP URL (`$config['soap_url']`) you have specified in `ispconfig3_account/config/config.inc.php` directly and see if it shows some error.

And, of course, have a look at the log files of Apache/nginx. Most likely, the error log will tell you the reason behind the message you are being shown.

**Comment by bhenstra** [comment]
Thanks for your help! It's fixed. I did check the Apache2 error log and it showed me this:

`[Sat Jan 25 22:04:39.692655 2020] [authz_core:error] [pid 20513] [client 127.0.0.1:40136] AH01630: client denied by server configuration: /var/www/ispconfig/remote/index.php
`
I did have a .htaccess file in /var/www/ispconfig with the following:

```
ErrorDocument 403 https://www.startpage.com/
<RequireAll>
    Require ip x.x.x.x y.y.y.y
</RequireAll>
```
Where "x.x.x.x" and "y.y.y.y" are IP addresses.

My fault... sorry. Thanks for your help.
